.github/workflows/ci.yml

```diff
@@ -4,82 +4,101 @@ on:
   push:
     branches: [ master ]
   pull_request:
-    branches: [ master ]
 
 jobs:
-  build:
```

Comment:
This looked like an early-github-actions experiment: I replaced it with a more common job that uses `actions/cache@v2`